src/layer/vector/Polyline.js

```diff
@@ -165,11 +171,17 @@ export var Polyline = Path.extend({
 	// the polyline in case of a multi-polyline, but can be overridden by passing
 	// a specific ring as a LatLng array (that you can earlier access with [`getLatLngs`](#polyline-getlatlngs)).
 	addLatLng: function (latlng, latlngs) {
```

Comment:
Suggestion to rename the second argument to `ring` because of the doc `passing a specific ring`

Reply:
`ring` itself is `latlngs` array

Reply:
Yes I know, but every time I read `addLatLng: function (latlng, latlngs)` I think first that I need to pass the complete latlngs of the polyline. It would be much clearer if `latlngs` is renamed to `ring`.

Reply:
> It would be much clearer if `latlngs` is renamed to `ring`.

I have no opinion.
@mourner @IvanSanchez @jonkoops

Reply:
Seems fine to me